src/Rest/GuildSticker.php

```diff
@@ -68,16 +68,13 @@ public function get(string $guildId, string $stickerId)
      */
     public function create(string $guildId, StickerBuilder $stickerBuilder)
```

Comment:
Add return type (not sure what the return type is though)

Reply:
[4be6233](https://github.com/dc-Ragnarok/Fenrir/pull/2/commits/4be62336d1b8d46bc214e776f7b7c18957951293)